tests/unit/apps/test_app.py

```diff
@@ -440,10 +440,11 @@ def test_upgrade_plan_application_already_disable_action_managed(status, config)
         "Change charm config of 'my_keystone' 'openstack-origin' to 'cloud:focal-victoria'",
         "Check if the workload of 'my_keystone' has been upgraded",
     ]
-    assert upgrade_plan.description == "Upgrade plan for 'my_keystone' from: ussuri to victoria"
+    assert upgrade_plan.description == "Upgrade plan for 'my_keystone' to victoria"
     assert_plan_description(upgrade_plan, steps_description)
 
 
+@unittest.skip("subordinates are removed")
```

Comment:
Shouldn't we remove this unit test instead of skipping it?

Reply:
I can remove it after pr #42 is merged.

Reply:
I don't understand why we need to wait for #42 to be merged before removing this test since subordinates-related changes should be handled in this PR.

Even if we decide to leave the test here for now, the skip message is really confusing; should say something like "subordinate check is removed" or "added supports for some subordinates" etc

Reply:
removed